modules/virt-cluster-resource-requirements.adoc

```diff
@@ -49,16 +53,21 @@ Calculate the cluster processor overhead requirements for {VirtProductName} by u
 .Cluster CPU overhead
 
 ----
-CPU overhead for infrastructure nodes ≈ 4
+CPU overhead for infrastructure nodes ≈ 4 cores
 ----
 
+{VirtProductName} increases the overall utilization of cluster level services such as logging, routing, and monitoring. To account for this workload, ensure that your infrastructure nodes have capacity allocated for 4 additional cores (4000 millicores).
+
 ----
-CPU overhead for worker nodes ≈ 2 + CPU overhead per virtual machine
+CPU overhead for worker nodes ≈ 2 cores + CPU overhead per virtual machine
 ----
 
+Worker nodes must have capacity allocated for 2 additional cores (2000 millicores) in addition to the CPUs required for virtual machine workloads.
```

Comment:
I'm stumbling over "must have capacity allocated" which appears a few times. Could we put this into the active voice?